python/ray/ray_constants.py

```diff
@@ -319,6 +319,11 @@ def to_memory_units(memory_bytes, round_up):
 # NOTE: This is equal to the C++ limit of (RAY_CONFIG::max_grpc_message_size)
 GRPC_CPP_MAX_MESSAGE_SIZE = 100 * 1024 * 1024
 
+# GRPC options
+GLOBAL_GRPC_OPTIONS = (("grpc.enable_http_proxy",
+                        os.environ.get("RAY_grpc_enable_http_proxy",
+                                       "0").lower() in ("1", "true")), )
```

Comment:
My small interest is maintaining monitor.py

Nit: previously the value of the option defaulted to the int `0`. Now it defaults to the bool `False`. I bet these values lead to the same behavior. To be very conservative, can we make sure it still defaults to `0`?

Reply:
Good catch! I didn't mean to use `False`. Sounds like adding a test case is needed.